WC_Layers/config.ini

```diff
@@ -0,0 +1,43 @@
+[general]
+# Specify whether this problem uses structured or unstructured mesh
+structured: False
+# Specify whether this problem uses sub-voxel r2s
+sub_voxel: False
+
+[step1]
+# Path to MCNP MESHTAL file containing neutron fluxes or a DAG-MCNP5
+# unstructured mesh tally .h5m file.
+meshtal: Mesh_with_Tally.h5m
+# Tally number within the meshtal file containing the fluxes for activation.
+tally_num: 2
```

Comment:
Is this relevant for an unstructured mesh?  It doesn't seem like your script to convert from OpenMC to H5M preserves any tally number (which is fine if we don't actually need it).

Reply:
No, it's not relevant here--only data from one tally is written to h5m, so the same ALARA input file is printed regardless of which number is chosen here